core/src/components/button/test/a11y/button.e2e.ts

```diff
@@ -1,6 +1,74 @@
+import AxeBuilder from '@axe-core/playwright';
 import { expect } from '@playwright/test';
 import { configs, test } from '@utils/test/playwright';
 
+configs({ directions: ['ltr'], themes: ['light', 'dark'] }).forEach(({ title, config }) => {
+  test.describe(title('button: a11y for ion-color()'), () => {
+    test('should not have accessibility violations', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-button>Default</ion-button>
+        <ion-button fill="solid">Solid</ion-button>
+        <ion-button fill="outline">Outline</ion-button>
+        <ion-button fill="clear">Clear</ion-button>
+      `,
+        config
+      );
+
+      const results = await new AxeBuilder({ page }).analyze();
+      expect(results.violations).toEqual([]);
+    });
+
+    test('focused state should not have accessibility violations', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-button class="ion-focused" fill="solid">Solid</ion-button>
+        <ion-button class="ion-focused" fill="outline">Outline</ion-button>
+        <ion-button class="ion-focused" fill="clear">Clear</ion-button>
+      `,
+        config
+      );
+
+      const results = await new AxeBuilder({ page }).analyze();
+      expect(results.violations).toEqual([]);
+    });
+
+    test('button in toolbar should not have accessibility violations', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-toolbar>
+          <ion-button fill="outline" class="ion-activated"> Start </ion-button>
```

Comment:
```suggestion
          <ion-button fill="outline" class="ion-activated">Start</ion-button>
```

Nit: Just to keep it consistent with the other tests 😄